<launch>
 <!-- RGBD camera -->
  <include file="$(find openni2_launch)/launch/openni2.launch">
	<arg name="depth_registration" value="true" />
    <arg name="publish_tf" value="false"/>
 </include>

 <!-- Primensense TF -->
  <include file="$(find object_detection)/launch/primesense_tf.launch"/>

 <!-- Transform publisher -->     
  <node pkg="object_detection" type="tf_cam_robot" name="tf_cam_robot_publisher" respawn="true" output="screen"/>   

 <!-- HSV tuning -->
  <node pkg="object_detection" type="hsv_tuning" name="hsv_tuning" respawn="true" output="screen"/>          



</launch>
